.flow-coverage-config

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
IMO would be nice to avoid another top level dotfile if we can.
I'd move it to `scripts/flow/coverage-config`.

Reply:
I'm not sure how to prevent Flow from trying to parse the JSON file if I move it here. Even adding it to the `[ignore]` block in the `.flowconfig` isn't sufficient.

Reply:
For now I'm just going to leave this as a dotfile. I don't know how to make Flow ignore it (and not error) otherwise, and it doesn't seem pressing enough to invest more than a few minutes into.